core/frontend/src/components/app/PirateModeMenu.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <v-card
+    class="d-flex flex-column align-center pa-3"
+    outlined
+    width="300"
+  >
+    <v-alert
+      v-if="!settings.is_pirate_mode"
+      colored-border
+      type="info"
+      icon="mdi-skull-crossbones"
+      elevation="2"
+      text
+    >
+      Use Pirate Mode to show hidden pages and advanced settings.
+      Note that it's an advanced mode and should be used with care.
```

Comment:
Seems a bit verbose. Maybe something like this?
```suggestion
      Pirate powers should be used with care.
```

Reply:
Done.